crates/sui-framework/sources/id.move

```diff
@@ -59,6 +59,12 @@ module sui::id {
         version: u64
     }
 
+    /// An object ID representing an ID that has been transferred during this transaction.
+    /// It can be used as the parent ID target for transferring child objects.
```

Comment:
I don't understand this sentence.

Reply:
Me neither. 

I think I was trying to write something like, "ya use it for transfer_to_object_id" but failed to write that coherently. Open to suggestions